src/linear_algebra/basis.lean

```diff
@@ -828,6 +828,19 @@ def equiv_of_is_basis {v : ι → M} {v' : ι' → M'} {f : M → M'} {g : M' 
     λ y, congr_arg (λ h:M' →ₗ[R] M', h y) this,
   ..hv.constr (f ∘ v) }
 
+/-- Isomorphism between the two modules, given two modules `M` and `M'` with respective bases
+`v` and `v'` and a bijection between the indexing sets of the two bases. -/
+def equiv_of_is_basis' {v : ι → M} {v' : ι' → M'} (hv : is_basis R v) (hv' : is_basis R v')
```

Comment:
Intuitively, I would say this is the more useful version, and the other one should get the `'`.
But I don't care too much. What do you think?

Reply:
Sounds good. I have exchanged the names.